app/views/home/admin.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <div class="row">
+    <div class="col-xs-3 col-sm-3 col-md-4 col-lg-3">
+      <a href="#">users</a>
+    </div>
+    <div class="col-xs-9 col-sm-9 col-md-8 col-lg-9">
+      <table class="table table-striped">
+        <% @users = User.all %>
```

Comment:
Move this variable assignment to controller. Variables initializations should be in controller.